src/renderer/renderScene.ts

```diff
@@ -598,6 +597,43 @@ export const renderSceneThrottled = throttleRAF(
   { trailing: true },
 );
 
+/** renderScene throttled to animation framerate */
+export const renderSceneThrottled = (
+  elements: readonly NonDeletedExcalidrawElement[],
+  appState: AppState,
+  selectionElement: NonDeletedExcalidrawElement | null,
+  scale: number,
+  rc: RoughCanvas,
+  canvas: HTMLCanvasElement,
+  renderConfig: RenderConfig,
+  callback?: (data: ReturnType<typeof renderScene>) => void,
+  throttle = true,
```

Comment:
shouldn't this be `false` ?

Reply:
why is it confusing? You'd expect that a function `renderThrottled` will throttle by default, no?

Reply:
yeah but then it shouldn't receive a param `throttle` since default behaviour is throttling, rather it should receive a param `disableThrottle` as thats where a param is needed